### DISCUSSION TITLE
Custom build steps

### DISCUSSION THREAD

**Discussion by avazquez-bit** [NativePHP/laravel#354] [Ideas]
Is there s way to increase app performance???
I understand it is executed over many programs, frameworks and other things, but sometimes is too slow in excecution, and sometines is faster but not as faster as I wish, my laravel app use blade icons and with this app I use to cache icons, to make sites faster and by this way app performance increase a lot, charge diferences from 10 senconds even above to 1 second or less it is awesome.

What Im asking is if there is a way to custom some part of build steps to cache blade icons or if there is a way to optimize app performance, thaks

**Comment by simonhamp** [comment]
> What Im asking is if there is a way to custom some part of build steps to cache blade icons

I'm going to translate this a little further for you: you would like to add custom commands to the build process because in your case you're using a library which creates a performance penalty in your application which you can avoid with an explicit custom build step.

Does this sound right?

**Reply by avazquez-bit** [reply]
Yeah, it is a ver good option, sorry by the late

**Reply by simonhamp** [reply]
Ok, I've renamed this discussion and moved it to the Ideas category to reflect this understanding.
